<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="pageValue">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>8</y>
     <width>47</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="pageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pages</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="scoreWindow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>571</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="zoomInBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="zoomOutBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
